src/template_helpers.rs

```diff
@@ -131,12 +137,22 @@ fn to_array_helper(v: &JsonValue) -> JsonValue {
     .into()
 }
 
+fn get_site_prefix() -> String {
+    let app_config = app_config::load();
```

Comment:
Maybe the prefix should instead be inserted at the top level rendering context ?
Or, more easily, registered together with the helpers, but as a partial instead of a helper? So that the value isn't recomputed every time.

Reply:
I will look into partials.